Client/Frontend/Intro/IntroScreenWelcomeView.swift

```diff
@@ -100,7 +100,7 @@ class IntroScreenWelcomeView: UIView, CardTheme {
     var signUpClosure: (() -> Void)?
     var signInClosure: (() -> Void)?
     // Basic variables
-    private var currentPage = 0
+    private var currentPage: Int32 = 0
```

Comment:
I needed to give this an explicit type in order for the downcasting from `Any` to work correctly when recording the event.